assets/report.json

```diff
@@ -3911,12 +3899,10 @@
     "flash.net": {
       "summary": {
         "max_points": 5,
-        "impl_points": 4,
+        "impl_points": 5,
         "stub_penalty": 0
       },
-      "missing": [
-        "getClassByAlias()"
-      ],
+      "missing": [],
       "stubbed": []
```

Comment:
There should be stubs here- sendToURL and registerClassAlias should be marked as stubs.

Reply:
Interesting, I'm guessing since `flash.net` isn't a class exactly https://github.com/Dinnerbone/ruffle/commit/7b5372cb2d7c442fd52b2c2b71c633c22d5e393f doesn't actually check for stubs in it. Didn't carefully check the code to see if that's true though.